terraform/environments/analytical-platform-compute/lakeformation-permissions.tf

```diff
@@ -19,3 +20,45 @@ resource "aws_lakeformation_permissions" "cadet_all_data" {
     }
   }
 }
+
+resource "aws_lakeformation_lf_tag" "domain" {
+  for_each = try(local.environment_configuration.cadet_lf_tags, {})
+  key = each.key
+  values = each.value
+}
+
+resource "aws_lakeformation_permissions" "cadet_domain_database_data" {
+  for_each = try(local.environment_configuration.cadet_lf_tags, {})
+
+  principal   = module.copy_apdp_cadet_metadata_to_compute_assumable_role.iam_role_arn
+  permissions = ["ALL"] # https://docs.aws.amazon.com/lake-formation/latest/dg/lf-permissions-reference.html
+
+  lf_tag_policy {
+    resource_type = "DATABASE"
+    expression {
+      key    = "domain"
+      values = each.value
+    }
+  }
+}
+
+resource "aws_lakeformation_permissions" "cadet_domain_table_data" {
+  for_each = try(local.environment_configuration.cadet_lf_tags, {})
+
+  principal   = module.copy_apdp_cadet_metadata_to_compute_assumable_role.iam_role_arn
+  permissions = ["ALL"] # https://docs.aws.amazon.com/lake-formation/latest/dg/lf-permissions-reference.html
+
+  lf_tag_policy {
+    resource_type = "TABLE"
+    expression {
+      key    = "domain"
+      values = each.value
+    }
+  }
+}
+
+import {
```

Comment:
Are dynamic references supported in import? I'm sure I've tried and failed at this before

Reply:
Merge into main and let's see.